app/Jobs/FetchNewJobs.php

```diff
@@ -61,8 +63,22 @@ public function handle(): void
         }
 
         // Fire the event
-        // @TODO: Apply filters here
-        $jobs = JobPost::visible()->unnotified()->orderBy('published_at', 'desc')->get()->all();
+        $this->notifyJobsPosted();
```

Comment:
If we're not doing anything with the $posts array then we should probably get rid of it now